packages/devtools_app/lib/main.dart

```diff
@@ -43,6 +44,11 @@ void main() async {
   setGlobal(DevToolsExtensionPoints, ExternalDevToolsExtensionPoints());
   setGlobal(IdeTheme, getIdeTheme());
 
+  // inspectorPreferences depends on the IdeTheme global
+  // to initialize
+  final inspectorPreferences = InspectorPreferencesController();
```

Comment:
Why inspectorPreferences are defined separately? We have PreferencesController, defined above. Can it include needed preferences?

Reply:
It definitely can.
Since the setting only concerns the inspector tab and no other parts of devtools, I thought it might be more appropriate the have it sequestered.
(I know they are saved in the same file though, so this was more meant as an interface).

If you think this should just be in preferences then I can definitely make an interface there :)

Reply:
Makes sense.